package.json

```diff
@@ -12,7 +12,7 @@
     }
   },
   "scripts": {
-    "build": "esbuild source.js --bundle --format=esm --outfile=index-esm.js",
+    "build": "esbuild source.js --bundle --format=esm --outfile=index-esm.js --platform=node",
```

Comment:
NOTE: `--platform=node` was necessary since `esbuild` didn't recognize `node:url` without it.